.circleci/config.yml

```diff
@@ -81,7 +81,7 @@ jobs:
 
     - run:
         name: Set Up Monitoring
-        command: node src/bin.js update https://adobeioruntime.net/api/v1/web/helix/helix-services/`node -e "console.log(require('./package.json').name.replace('@adobe/helix-', ''))"`@v`node -e "console.log(require('./package.json').version.match(/^[0-9]+.[0-9]+/)[0])"`/_status_check/healthcheck.json
+        command: node src/newrelic.js setup https://adobeioruntime.net/api/v1/web/helix/helix-services/`node -e "console.log(require('./package.json').name.replace('@adobe/helix-', ''))"`@v`node -e "console.log(require('./package.json').version.match(/^[0-9]+.[0-9]+/)[0])"`/_status_check/healthcheck.json `node src/statuspage.js setup --silent`
```

Comment:
@trieloff: I'm not sure if the `v` is correct here, helix services seem to get deployed without it, e.g.
```
https://adobeioruntime.net/api/v1/web/helix/helix-services/status@4.3.5/_status_check/healthcheck.json
```

Reply:
`@vX` and `@vX.Y` is for the major/minor symlinks.